scripts/ci

```diff
@@ -7,8 +7,3 @@ cd $(dirname $0)
 ./validate
 ./test -cover
 ./package
-
```

Comment:
Why remove those?

Reply:
> Why remove those?

Because:
- `./bin/latest_image` is no longer being created by the `scripts/package` script.
- `latest_image` no longer appears in this repo.
- A missing file will cause this script to fail.
- Finally, if these `echo` statements are actually important, a comment should be present in this file explaining their importance.